reviewers.js

```diff
@@ -45,8 +53,10 @@ function getReviewerByName(reviewerName, reviewers) {
  ****************************************************************/
 function getReviewerByReviewTitle(reviewTitle, reviewers) {
   //TODO: ADD YOUR CODE HERE
+  return reviewers.find((value)=>
+  value.reviewTitle=reviewTitle);
```

Comment:
There's no property in the object called `reviewTitle`, you're supposed to get the `title` of the book from the `book` object